install-macos.sh

```diff
@@ -68,6 +70,29 @@ else
     exit 1
 fi
 
+##
+# Check Developer ID
+##
+printf "\033[34m\n* Ensuring package Developer ID matches\n\033[0m"
+
+if pkgutil --check-signature $PKG_PATH | grep -q "$DEVELOPER_ID"; then
+    printf "\033[34mDeveloper ID matches.\n\033[0m"
+else
+    printf "\033[31m Developer ID does not match. Please contact support@vanta.com \033[0m\n"
+    exit 1
+fi
+
+##
+# Check Developer Certificate Fingerprint
+##
+printf "\033[34m\n* Ensuring package Developer Certificate Fingerprint matches\n\033[0m"
+if pkgutil --check-signature $PKG_PATH | tr -d '\n' | tr -d ' ' | grep -q "SHA256Fingerprint:$CERT_SHA_FINGERPRINT"; then
```

Comment:
Full path to grep, and might as well full path to tr as well here.

Tested locally – pkgutil is pretty snappy so I don't have a problem with running the same command twice instead of saving the output in an intermediate variable.

Reply:
Updated 👍